fix(kong) wait for (#33)
Kong/kong-collector-helm: README.md

To install the chart with the release name `my-release`:

> 1. When Kong, make sure that admin_gui_flags includes an "IMMUNITY_ENABLED" set to true.  An example of this can be found in kong-values.yaml file in this repo.  This does have to be done on Kong installation, and it is necessary to see the Alerts page on Kong Manager.

Reviewer: ```suggestion
1. When installing Kong, make sure that `admin_gui_flags` includes the `IMMUNITY_ENABLED` flag set to `true`. This must be done on Kong installation, and it is necessary to see the Alerts page on Kong Manager.
    
    You can find an example of this in the [kong-values.yaml](https://github.com/Kong/kong-collector-helm/blob/master/kong-values.yaml) file in this repo. 
```

secret/kong-admin-token-secret created
```

> 3. Set up collector, overriding Kong Admin host, servicePort and token to ensure Kong Admin API is reachable by collector.

Reviewer: ```suggestion
3. Set up collector, overriding Kong Admin host, servicePort, and token to ensure Kong Admin API is reachable by collector.
```
